server/lib/notifications.js

```diff
@@ -236,9 +238,77 @@ const notifyUpdateSubscribers = async activity => {
   const emailOpts = { from: activity.data.fromEmail };
   const update = await models.Update.findByPk(activity.data.update.id);
   const allUsers = await update.getUsersToNotify();
-  return notifySubscribers(allUsers, activity, emailOpts);
+  const modifiedActivity = replaceVideosByImagePreviews(activity);
+  return notifySubscribers(allUsers, modifiedActivity, emailOpts);
 };
 
+function replaceVideosByImagePreviews(activity) {
+  const sanitizeOptions = buildSanitizerOptions({
```

Comment:
Would be great to import this from https://github.com/opencollective/opencollective-api/blob/0bc1022162162c6b2621eb6c6df8e897145ae93f/server/models/Update.js#L15, to make sure changes to options there are added here as well.

Reply:
Corrected this one. 👍🏽